drivers/pwm/pwm_ite_it8xxx2.c

```diff
@@ -106,6 +106,11 @@ static int pwm_it8xxx2_set_cycles(const struct device *dev,
 		*reg_pwmpol &= ~BIT(ch);
 	}
 
+	/* Enable PWM output open-drain */
+	if ((flags >> PWM_IT8XXX2_FLAGS_SHIFT) & PWM_IT8XXX2_OPEN_DRAIN_MASK) {
```

Comment:
You can just do `if (flags & PWM_IT8XXX2_OPEN_DRAIN)` and drop the mask constant.

Reply:
Done.